Enable advanced power menu by default

@westcripp

https://github.com/ResurrectionRemix/android_frameworks_base/commit/6e3dc0b2aa473ce417dca77130878adce3d0f8b6

project frameworks/base/
diff --git a/frameworks/base/services/core/java/com/android/server/power/ShutdownThread.java b/frameworks/base/services/core/java/com/android/server/power/ShutdownThread.java
index c92757c4b58..c487af0e392 100644
--- a/frameworks/base/services/core/java/com/android/server/power/ShutdownThread.java
+++ b/frameworks/base/services/core/java/com/android/server/power/ShutdownThread.java
@@ -174,7 +174,7 @@ public final class ShutdownThread extends Thread {
         KeyguardManager km = (KeyguardManager) context.getSystemService(Context.KEYGUARD_SERVICE);
         boolean keyguardLocked = km.inKeyguardRestrictedInputMode() && km.isKeyguardSecure();
         boolean advancedRebootEnabled = MKSettings.Secure.getInt(context.getContentResolver(),
-                MKSettings.Secure.ADVANCED_REBOOT, 0) == 1;
+                MKSettings.Secure.ADVANCED_REBOOT, 1) == 1;
         boolean isPrimaryUser = UserHandle.getCallingUserId() == UserHandle.USER_OWNER;
 
         return advancedRebootEnabled && !keyguardLocked && isPrimaryUser;
